resources/views/components/wallet/overview/wallet.blade.php

```diff
@@ -12,7 +12,7 @@
 <x-wallet.overview.item :title="trans('general.overview')">
     <x-wallet.overview.item-entry
         :title="trans('pages.wallet.name')"
-        :value="$wallet->walletName()"
+        :value="$wallet->usernameBeforeKnown()"
```

Comment:
you'll want the "known name" to always overrule on-chain ones as we only set this for specific reasons (e.g. to highlight legit accounts, or to highlight someone who abuses the usernames)